setup.py

```diff
@@ -103,6 +103,9 @@
         'Topic :: Internet :: WWW/HTTP :: WSGI :: Server',
     ],
     entry_points={
+        'console_scripts': [
+            'cheroot = cheroot.__main__:main',
```

Comment:
I'd want to move the implementation code to cli module and then reuse it also in `__main__`, because architecturally speaking both `__main__` and `console_scripts` are the entry points.